csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -565,8 +565,8 @@ The properties are: `names`, `organization`, `summary`, and `urls`.
 
 #### 3.1.1.1 Acknowledgments Type - Names
 
-List of acknowledged names (`names`) has value type `array` with 1 or more items holds the names of entities being recognized.
-Every such item of value type `string` with 1 or more characters represents the name of entity being recognized and contains the name of a single person.
+List of acknowledged names (`names`) has value type `array` with 1 or more items holds the names of persons being recognized.
+Every such item of value type `string` with 1 or more characters represents the name of person being recognized and contains the name of a single person.
```

Comment:
I would add a `the` and would also consider the term `contributor` - that would in my perception nicely include any natural or _juristical_ (as we say in German, lacking a translation here) person. Shortly, any entity that can be named.

Reply:
I added the `the`. I did not change person to contributor (yet) as organizations should be listed in `organization`. If I'm not missing anything (right now), then there shouldn't be a need to cover legal persons (even though they are also persons...)